packages/client/hmi-client/src/temp/StratifyPlayground.vue

```diff
@@ -0,0 +1,568 @@
+				);
+				//TODO
+				console.log(outputModel);
+			} catch (e: any) {
+				console.error(e.message);
+			}
+			d3.select('#solution').text('Hello'); //JSON.stringify(output, null, 2));
+		},
+		// Used to create sample models for stratifying tests
+		// Will not be requried in the long run as we will be moving to storing these in DB
+		async spawnModelA(e) {
+			modelA = petrinets[e.target.value];
+			g = await parsePetriNet2IGraph(modelA);
+			g = runDagreLayout(_.cloneDeep(g));
+			this.refresh();
+			this.jsonOutput();
+		},
+		clearA() {
+			modelA = { T: [], S: [], I: [], O: [] };
+			g = { width: 500, height: 500, nodes: [], edges: [] };
+			g = runDagreLayout(_.cloneDeep(g));
+			this.refresh();
+		},
+		loadOntology() {
+			let model = petrinets[3];
+			g2 = parsePetriNet2IGraph(model);
+			g2 = runDagreLayout(_.cloneDeep(g2));
+			this.refresh();
+			this.jsonOutput();
+			console.log(model);
+		},
+		typeModel() {
+			console.log('Start typing model');
+			//Model 1 =
+			//Ontology Model =
+			//Mapping =
+			//Api Call
+		},
+		async testStratify() {
+			console.log('Start stratify');
+			try {
+				const resp = await API.get(`model-service/models/testStratify`);
+				const output = resp.data;
+				console.log('Done Stratify');
+				//TODO
+				console.log(output);
+			} catch (e: any) {
+				console.error(e.message);
+			}
+		},
+		/*
+		TODO: https://github.com/DARPA-ASKEM/Terarium/issues/868
+		Inputs:
+			Petrinet 
+			Petrinet
+		Output:
+			(Graph/Petrinet)
+		*/
+		async blindStratification(modelA, modelB) {
+			let petrinetOne = petrinets[modelA]; //Hard code SIRD for now
+			let petrinetTwo = petrinets[modelB]; //Hard code QNQ for now
+
+			let graphOne = parsePetriNet2IGraph(petrinetOne);
+			let graphTwo = parsePetriNet2IGraph(petrinetTwo);
+			let resultGraph: IGraph<NodeData, EdgeData>;
+
+			resultGraph = this.cloneFirstGraph(graphOne, graphTwo);
+
+			//Add graphTwo's shape to connect everything
+			for (let i = 0; i < graphTwo.edges.length; i++) {
+				for (let j = 0; j < graphOne.nodes.length; j++) {
+					if (graphOne.nodes[j].data.type == 'S') {
+						if (graphTwo.nodes[i].data.type == 'T') {
+							//Create new transition: (Example, inf,Q)
+							let newTransitionId = graphOne.nodes[j].id + ',' + graphTwo.nodes[i].id;
+							let newTransitionLabel = graphOne.nodes[j].label + ',' + graphTwo.nodes[i].label;
+							resultGraph.nodes.push({
+								id: newTransitionId,
+								label: newTransitionLabel,
+								data: { type: NodeType.Transition },
+								height: graphOne.nodes[j].height,
+								width: graphOne.nodes[j].width,
+								x: graphOne.nodes[j].x,
+								y: graphOne.nodes[j].y,
+								nodes: []
+							});
+						}
+						//Create this transition's edge(s)
+						let newSource = graphOne.nodes[j].id + ',' + graphTwo.edges[i].source;
+						let newTarget = graphOne.nodes[j].id + ',' + graphTwo.edges[i].target;
+						resultGraph.edges.push({ source: newSource, target: newTarget });
```

Comment:
This part looks a bit suspicious, that it being outside of the nested ifs.
Can you provide a small textual description for clarify?

Reply:
rereading this there is something weird going on with looking at edges vs nodes with the same iterator 
Give me a few seconds to see what I was thinking and comment it better or change it a tad